texts/Bade Bhaisaab (disambiguated) .txt

```diff
@@ -0,0 +1,1586 @@
+"<मेरे>"
+	"मैं" prn pers p1 mf sg gen
+"<भाई>"
+;	"भाई" n m sg nom
+	"भाई" n m sg obl
```

Comment:
This should be `nom`, not `obl`. `obl` is the oblique case used in situations like "भाई को", "किताबों में", etc where there is a 'suffix' after the noun, whereas `nom` is used for plain nouns. Please make sure this applies to all the nouns in this disambiguation.